src/imitation/scripts/common/train.py

```diff
@@ -102,7 +102,7 @@ def eval_policy(
         # Generate trajectories with the RL algorithm's env - SB3 may apply wrappers
         # under the hood to get it to work with the RL algorithm (e.g. transposing
         # images so they can be fed into CNNs).
-        train_env = rl_algo.get_env()
+        train_env = cast(vec_env.VecEnv, rl_algo.get_env())
```

Comment:
[`get_env`](https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/common/base_class.py#L440) should return an `Optional[VecEnv]`. Asserting it's non-None feels a bit more robust than a cast to me (so we actually do the type check at runtime), but it would require an additional LOC so not obviously worth it.